health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -0,0 +1,67 @@
+        </Card>,
+      ]}
+      onOverlayClick={onClose}
+      footerChildren={[
+        <Button
+          className={"campaign-type-alert-button"}
+          type={"button"}
+          size={"large"}
+          variation={"secondary"}
+          label={t(`HCM_MICROPLAN_EDIT_POPULATION_CLOSE`)}
+          onClick={onClose}
+        />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)